streaming_form_data/parser.py

```diff
@@ -10,10 +10,19 @@ class ParseFailedException(Exception):
 def parse_content_boundary(headers):
     content_type_header = 'Content-Type'
 
+    # try finding popular case forms
```

Comment:
I think you could simply remove this block when the dictionary is being looped on anyway.

Reply:
I thought about it before.
All tests will pass the same way without first block. I kept the first block for the performance. For most popular cases we will not degrade speed to linear search.

From the other hand I do not expect lots of headers. So we can ignore performance here.

So I would move decision about first block for you.
If you decide to keep it - we will keep. In other case I will update pull request.

Reply:
> From the other hand I do not expect lots of headers. So we can ignore performance here.

I agree with you here. It's probably simpler to just remove the first block.